chinese/chinese-cuisine.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
nice that you have 2 commits. If we make each page a separate commit, it would be even better